hw/ip/hmac/data/hmac.hjson

```diff
@@ -67,12 +67,28 @@
         }
         { bits: "2",
           name: "endian_swap",
-          desc: "Convert TL-UL wdata[31:0] to big-endian style {w[7:0], w[15:8], .. }",
+          desc: '''Endian swap.
+
+                If 0 each individual multi-byte value, regardless of its
```

Comment:
I wonder whether this should be "If 0 then...". Otherwise, it might be a bit difficult to break up the sentence (if there wasn't the subclause later, I'd suggest a comma: for the "If 1" case, either a comma or a "then" would probably be fine).

Reply:
Done ("If 0 then... If 1, ...").